main.go

```diff
@@ -41,7 +41,10 @@ func RootCmd() *cobra.Command {
 			kinesisStreamName, err := getKinesisStreamName(tags, kinesisStreamNameArg)
 			check(err, "kinesis stream name not found")
 
-			placeholders := map[string]string{"KINESIS_STREAM": kinesisStreamName}
+			systemdUnit, err := getSystemdUnit(tags)
+			check(err, "Missing tag - application log shipping will not be configured")
```

Comment:
I think `@guardian/cdk` should add this tag automatically, see https://github.com/guardian/amiable/pull/127 for more details.